google-cloud-storage/src/main/java/com/google/cloud/storage/MultipartUploadClientImpl.java

```diff
@@ -45,9 +48,20 @@ final class MultipartUploadClientImpl extends MultipartUploadClient {
     this.uri = uri;
   }
 
+  @Override
   @BetaApi
   public CreateMultipartUploadResponse createMultipartUpload(CreateMultipartUploadRequest request)
       throws IOException {
     return httpRequestManager.sendCreateMultipartUploadRequest(uri, request, options);
   }
+
+  @Override
+  @BetaApi
+  public ListPartsResponse listParts(ListPartsRequest request) throws IOException {
+
+    return retrier.run(
+        Retrying.alwaysRetry(),
```

Comment:
This should be `options.getRetryAlgorithmManager().idempotent()`.

Retrying.alwaysRetry() is primarily for tests.

Reply:
Done.